daprblog/content/posts/2023/fuzzing-audit.md

```diff
@@ -0,0 +1,47 @@
+---
+date: "2023-06-30T07:00:00-07:00"
+title: "Dapr completes fuzzing audit"
+linkTitle: "dapr-fuzzing-audit"
+author: AdamKorcz
```

Comment:
Suggest to change to "Dapr project maintainers" unless this needs to come from you